src/Logger.php

```diff
@@ -109,16 +105,30 @@ public function __construct(array $targets = [])
     /**
      * Returns the text display of the specified level.
      *
-     * @param mixed $level the message level, e.g. {@see LogLevel::ERROR}, {@see LogLevel::WARNING}.
-     *
-     * @return string the text display of the level
+     * @param mixed $level The message level, e.g. {@see LogLevel::ERROR}, {@see LogLevel::WARNING}.
+     * @return string The text display of the level.
+     * @throws InvalidArgumentException for invalid log message level.
      */
     public static function getLevelName($level): string
     {
-        if (is_string($level)) {
-            return $level;
+        if (!is_string($level)) {
+            throw new InvalidArgumentException(sprintf(
+                'The log message level must be a string, %s received.',
+                gettype($level)
+            ));
+        }
+
+        $level = strtolower($level);
```

Comment:
Do we really need to support non-lowercased levels? I think we can remove it.

Reply:
Ok.